CMakeLists.txt

```diff
@@ -51,6 +50,9 @@ ENDIF()
 # ${OpenMVS_BINARY_DIR}.
 PROJECT(OpenMVS)
 
+cmake_policy(SET CMP0127 NEW)
+cmake_dependent_option(OpenMVS_USE_SSE "Enable SSE optimizations" ON "NOT CMAKE_SYSTEM_PROCESSOR MATCHES \"^(arm|ARM|aarch64|AARCH64).\"" OFF)
```

Comment:
Doesn't this implicitly raise the minimum required version to CMake 3.22?
Other policies are handled with guards in lines 7 to 15.
And the feature enabled by CMP0127 simples doesn't exist before CMake 3.22,  
https://cmake.org/cmake/help/latest/policy/CMP0127.html

Reply:
thx for spotting this; can you modify the if to not require the new policy? if not lets update cmake version

Reply:
It should be enough to put the condition result into a variable, and use this variable value instead of the original `ON`.